### PR TITLE
fixes #24382 - increase repo password length

### PR DESCRIPTION
**Pull request by thomasmckay**

### CODE REVIEW

File: db/migrate/20180725083800_change_repository_password_length.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class ChangeRepositoryPasswordLength < ActiveRecord::Migration[5.1]
+  def change
+    change_column :katello_repositories, :upstream_password, :string, limit: 1024
```
Comment (sean797) [inline db/migrate/20180725083800_change_repository_password_length.rb]: The issue mentions `My example generates a key 760 characters.` should we consider setting this a little higher than `1024` ? or using text?

Comment (thomasmckay) [reply db/migrate/20180725083800_change_repository_password_length.rb]: @sean797 - confirmed that 1024 is max. I'm alright changing it to _text_, if you think that would be better but may be overkill.

Comment (thomasmckay) [reply db/migrate/20180725083800_change_repository_password_length.rb]: edit: let me just change to _text_ so there's never a worry

Comment (sean797) [reply db/migrate/20180725083800_change_repository_password_length.rb]: I wasn't sure what the max was, I couldn't find an RCF or any kind of docker API spec so wasn't sure.

If `1024` is the max, then I'm okay with `:string`, I think `:string` is more efficient for stuff like this anyway.

**Pull request merged**